### PR TITLE
Update QUBO: construct QUBO from graph

### PR DESCRIPTION
**Pull request by SciCodePhy**
This is an update of QUBO problem #54 :
* I add a function which can construct QUBO from a simple graph. The inputs are: a simple graph and a weights vector.

Maybe we can further construct QUBO directly from simple weighted graph as suggested by @hmyuuu (to make the interface more robust?). But I am not sure whether this would be redundant.

### CODE REVIEW

**Comment by hmyuuu** [comment]
Maybe a function that can generate  a simple weighted graph from QUOB.matrix is enough, I guess?

**Comment by SciCodePhy** [comment]
Decision: this function is enough.

File: src/ProblemReductions.jl [hunk 1]
```diff
@@ -24,7 +24,7 @@ export IndependentSet
 export VertexCovering, is_vertex_covering, vertex_covering_energy
 export SetPacking, is_set_packing
 export DominatingSet
-export QUBO
+export QUBO, QUBO_from_SimpleGraph
```
Comment (GiggleLiu) [inline src/ProblemReductions.jl:27]: Maybe we can use the QUBO as a constructor.

File: src/models/QUBO.jl [hunk 2]
```diff
@@ -15,6 +15,19 @@ struct QUBO{QT<:AbstractMatrix} <: AbstractProblem
     end
 end
 Base.:(==)(a::QUBO, b::QUBO) = a.matrix == b.matrix
+function QUBO_from_SimpleGraph(graph::SimpleGraph, weights::Vector)
```
Comment (GiggleLiu) [inline src/models/QUBO.jl:18]: Please also add onsite terms.

**Pull request merged**